google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -117,6 +118,19 @@
   ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference[] documentReferences, @Nullable FieldMask fieldMask);
 
+  /**
+   * Retrieves multiple documents from Firestore, while optionally applying a field mask to reduce
+   * the amount of data transmitted.
```

Comment:
Please mention that the documents will be returned out of order and that missing documents will not be returned at all.